haystack/preprocessor/utils.py

```diff
@@ -19,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 
-def eval_data_from_file(filename: str) -> Tuple[List[Document], List[Label]]:
+def eval_data_from_file(filename: str, n_docs: Union[int, bool]=None) -> Tuple[List[Document], List[Label]]:
```

Comment:
Let's rename `n_docs` to  `max_docs` and add a doc string :)

Reply:
Fixed in latest commit